drivers/timer/apic_tsc.c

```diff
@@ -209,6 +209,7 @@ uint64_t sys_clock_cycle_get_64(void)
 {
 	return rdtsc();
 }
+EXPORT_SYMBOL(sys_clock_cycle_get_64);
```

Comment:
I think this sort of thing would be better served in drivers/timer/export.c that optionally is built with LLEXT enabled

Since there's a few timer API symbols that we might want to export, in particular cycle_get_32 and cycle_get_64 (should be checked for availability!)

Reply:
Done.

Reply:
Then there is of course the third option of exporting only the intel adsp implementation, because that is all I need.

Reply:
@jsarha you could also add a `__weak()` version of the function, not sure if that would be preferred.

Reply:
Thanks, now there is a new version with a __weak dummy implementation of sys_clock_cycle_get_64().

Reply:
```c
#ifdef CONFIG_TIMER_HAS_64BIT_CYCLE_COUNTER
EXPORT_SYMBOL(sys_clock_cycle_get_64());
#endif
```

Reply:
Thanks. It was not hard to find, but I did not know what to look for. Hope this is good now.